seahorn/jobs2/byte_buf_eq2/aws_byte_buf_eq_harness2.c

```diff
@@ -0,0 +1,55 @@
+/*
+ *
+ */
+
+#include <aws/common/byte_buf.h>
+#include <byte_buf_helper.h>
+#include <seahorn/seahorn.h>
+#include <utils.h>
+
+extern bool sea_is_modified(char *);
+extern void sea_tracking_on();
+extern void sea_tracking_off();
+extern void sea_reset_modified(char *);
+
+int main() {
+  /* parameters */
+  struct aws_byte_buf lhs;
+  initialize_byte_buf(&lhs);
+  struct aws_byte_buf rhs;
+  initialize_byte_buf(&rhs);
+
+  /* assumptions */
+  assume(aws_byte_buf_is_valid(&lhs));
+  if (nd_bool()) {
+    rhs = lhs;
+  } else {
+    assume(aws_byte_buf_is_valid(&rhs));
+  }
+
+  sea_tracking_on();
+  sea_reset_modified((char *)&lhs);
```

Comment:
explicit reset is required? I was assuming that modified flag is 0 until tracking is enabled

Reply:
no not required... removed...